planning/behavior_path_planner/src/scene_module/pull_out/util.cpp

```diff
@@ -108,11 +108,6 @@ lanelet::ConstLanelets getPullOutLanes(
     return shoulder_lanes;
   }
 
-  // Get pull out lanes
-  lanelet::ConstLanelet current_lane;
-  lanelet::utils::query::getClosestLanelet(
-    current_lanes, planner_data->self_pose->pose, &current_lane);
-
```

Comment:
also the arg of current_lanes and
```
  if (current_lanes.empty()) {
    return shoulder_lanes;
  }
```
are not needed, but @purewater0901 wants to use them later with another PR, so leave them.